uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,55 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'leave-absences/shared/apis/entitlement-api',
+], function() {
+  'use strict';
+
+  describe('EntitlementAPI', function (){
+    var entitlementAPI;
+
+    beforeEach(module('leave-absences.apis'));
+
+    beforeEach( inject( function(_EntitlementAPI_) {
+      entitlementAPI = _EntitlementAPI_;
+    }));
+
+    it('has expected end points', function() {
+      expect(Object.keys(entitlementAPI)).toEqual(['all', 'breakdown']);
+    });
+
+    describe('gets all entitlements', function() {
```

Comment:
nested describe read like sentence so i will keep the sentence as well have method name